miniapps/imgur-upload/src/manifest.ts

```diff
@@ -0,0 +1,23 @@
+import { ModManifest } from "@mod-protocol/core";
+import action from "./action";
+import error from "./error";
+import loading from "./loading";
+import upload from "./upload";
+
+const manifest: ModManifest = {
+  slug: "imgur-upload",
+  name: "Upload image to Imgur",
```

Comment:
I wonder what the best approach here is; giving the user the choice whether to upload to Imgur or to IPFS feels like a choice that we should be forcing most users to make in this kind of interface. Not sure what the alternative is, but I guess for now its fine - could be a choice that the devs are making for now

Reply:
I think we should make it easier for clients to choose which miniapps to include (e.g. by name instead of just exposing an array) and also to allow them to insert their own outside of the registry

Reply:
they are able to insert their own today, will add to docs

Reply:
they can insert outside the registry, but we should add it to the docs

Reply:
rendering priority is done by order iirc; they can insert at the start or end, or remove the Mod that conflicts

re automatic distro: I think we will get there eventually, but I think safety/feeling of being in control is more important atm; not sure we should update over the wire, unless we do it via a json file where we give them the choice: latest or versioned

Reply:
if the user is going to be able to include their own Mods then they need to be able to also specify the API url to use... will need to think of a way to specify that

Reply:
I don't follow, wouldn't they hard code that?